cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -528,6 +529,52 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     clusterList.waitForPage();
   });
 
+  describe('Cluster Details Tabs', () => {
+    const tabbedPo = new TabbedPo('[data-testid="tabbed-block"]');
+    const clusterDetail = new ClusterManagerDetailImportedGenericPagePo(undefined, 'local');
+
+    beforeEach( () => {
+      HomePagePo.navTo();
```

Comment:
you can navigate directly to the cluster manager page using `ClusterManagerListPagePo.navTo();`, that way you can remove the below chunk of code

```
const burgerMenu = new BurgerMenuPo();

BurgerMenuPo.toggle();
const clusterManagementNavItem = burgerMenu.links().contains(`Cluster Management`);

clusterManagementNavItem.should('exist');
clusterManagementNavItem.click();
```

Reply:
just this last one ^ and we're good to go